agent/taskresource/envFiles/envfile.go

```diff
@@ -271,7 +272,7 @@ func (envfile *EnvironmentFileResource) setTerminalReason(reason string) {
 	})
 }
 
-// Create performs resource creation. This retrieves env file contents in parallel
+// Create performs resource creation. This retrieves env file contents concurrently
```

Comment:
We need to do the performance test testing downloading 10 S3 files at the same time before we release, in case there are some issues we didn't know about.

Reply:
will make a note of this to include as test case.